parts/before-main.php

```diff
@@ -77,5 +77,11 @@
 </span> Connect
 			</a>
 		</nav>
-		<div class="site-header-logo sh-item"><a href="<?php echo esc_url( home_url() ); ?>"><img src="<?php echo esc_url( get_stylesheet_directory_uri() . '/images/WSM_logo.svg' ); ?>" alt="Washington State Magazine"></a></div>
+		<div class="site-header-logo sh-item"><a href="<?php echo esc_url( home_url() ); ?>"><img src="<?php
+			if ( spine_has_featured_image() ) {
+				$black_banner = sanitize_html_class( get_post_meta( get_the_ID(), '_wsm_banner_color', true ) );
```

Comment:
We can probably leave out `sanitize_html_class()` here and just check if `$black_banner` is truthy since we aren't using the actual value.